selfservice/strategy/passkey/passkey_settings.go

```diff
@@ -156,7 +158,7 @@ func (s *Strategy) identityListWebAuthn(id *identity.Identity) (*identity.Creden
 
 func (s *Strategy) Settings(w http.ResponseWriter, r *http.Request, f *settings.Flow, ss *session.Session) (*settings.UpdateContext, error) {
 	if f.Type != flow.TypeBrowser {
-		return nil, flow.ErrStrategyNotResponsible
+		return nil, errors.WithStack(flow.ErrStrategyNotResponsible)
```

Comment:
Do we need this? I'm not sure if we correctly assert with `errors.Is` everywhere

Reply:
I checked, looks like we do.